docs/webhooks/events.md

```diff
@@ -254,21 +254,21 @@ Type | Description
 
 Property | Value | Description
 ---------|-------|------------
-`answers` | Hash | Survey question and responses
 `submitted_at` | date-time | Date/time response was submitted (EST/EDT or GMT -5/GMT -4)
-`response_id` | String | Survey external Response ID
-`url` | Hash | URL variable value for Survey
 `main_indicator` | Integer | Main question result
 `main_indicator_scaled` | Float | Main question result scaled to 0..1
 `intervention_id` | String | ID of the intervention related to this survey
 `survey_id` | String | ID of the survey
 `source_id` | String | ID of the source related to this survey
 `user_id` | String | ID of the user related to this survey
-`context_data` | Hash | Context_data is present only if the content has context_data associated. The context_data hash keys are the custom fields keys.
+`questions` | Array of Hashes | List of questions and their responses. Each entry contains two keys : `text` and `value`
+`response_foreign_id` | String | Internal Response ID fetched from the provider.
+`answers` | Hash | `[Deprecated]` Survey question and responses
+`foreign_id` | String | `[Deprecated]`. Internal Response ID generated by Engage Digital.
```

Comment:
```suggestion
`foreign_id` | String | `[Deprecated]` Please use `id` instead. Internal Response ID generated by Engage Digital.
```

Reply:
BTW you can probably do the same for `answers` -> `questions`

Reply:
Good call, done along with answers part in 726673c